content/docs/pipelines/tutorials/pipelines-tutorial.md

```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
 1. Select or create a project on the [GCP Console][gcp-console].
 1. Make sure that billing is enabled for your project. See the guide to
   [modifying a project's billing settings][billing-guide].
-1. Install the [Cloud SDK][cloud-sdk].
+1. Use Cloud Shell to grant your team access to Kubeflow by granting them **IAP-secured Web App User** roles:
```

Comment:
My understanding is that users need the Project Owner and IAP-secured Web App User roles to use Kubeflow on GKE. Project Owner can only be assigned in Cloud Console, so it probably makes sense to have them use Cloud Console. How about something like this?

1. Use [Cloud Console to grant your team access][grant-access] to Kubeflow by granting them access to the following roles:

    *  **Project Owner**: To ensure that your team can access all of the resources used in this guide, grant your team access to the **Project Owner** role.
    * **IAP-secured Web App User**: This guide uses Cloud Identity-Aware Proxy (IAP) to secure access to your Kubeflow cluster. Your team must be members of the **IAP-secured Web App User** role to authenticate with the Kubeflow web application.
 
[grant-access]: https://cloud.google.com/iam/docs/granting-changing-revoking-access#using_the

Reply:
That makes sense, I appreciate your pointing this out. I have updated the text with your suggestion and some mild edits as follows:

  1. Use [Cloud console to grant your team access](https://cloud.google.com/iam/docs/granting-changing-revoking-access#using_the) to Kubeflow _by assigning them the following roles_:
     
      - **Project Owner**: _Ensures that your team can access all of the resources used in this guide._
      - **IAP-secured Web App User**: This guide uses Cloud Identity-Aware Proxy (IAP) to secure access to your Kubeflow cluster. Your team must be members of the **IAP-secured Web App User** role to authenticate with the Kubeflow web application.

Thanks!